services/filesystem-provider/ej2-amazon-s3-aspcore-file-provider/Controllers/PdfViewerController.cs

```diff
@@ -0,0 +1,285 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Caching.Memory;
+using Syncfusion.EJ2.FileManager.AmazonS3FileProvider;
+using Newtonsoft.Json;
+using Syncfusion.EJ2.PdfViewer;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+
+namespace EJ2AmazonS3ASPCoreFileProvider.Controllers
+{
+    [Route("file-api/[controller]")]
+    [EnableCors("AllowAllOrigins")]
+    [ApiController]
+    public class PdfViewerController : ControllerBase
+    {
+        public AmazonS3FileProvider operation;
+        public IMemoryCache _mCache;
+
+        private IWebHostEnvironment _hostingEnvironment;
+
+        public PdfViewerController(IWebHostEnvironment hostingEnvironment, IMemoryCache cache)
+        {
+            _mCache = cache;
+            _hostingEnvironment = hostingEnvironment;
+            this.operation = new AmazonS3FileProvider();
+
+            string name = System.Environment.GetEnvironmentVariable("OBJECT_STORE_BUCKET");
+            string awsAccessKeyId = System.Environment.GetEnvironmentVariable("OBJECT_STORE_ACCESS_KEY_ID");
+            string awsSecretAccessKey = System.Environment.GetEnvironmentVariable("OBJECT_STORE_ACCESS_KEY");
+            string serviceName = System.Environment.GetEnvironmentVariable("OBJECT_STORE_HOST");
+            this.operation.RegisterAmazonS3(name, awsAccessKeyId, awsSecretAccessKey, serviceName);
+        }
+
+        [AcceptVerbs("Post")]
+        [HttpPost]
+        [Route("Load")]
+        [Authorize("View")]
+        public IActionResult Load([FromBody] Dictionary<string, string> jsonObject)
+        {
+            Console.WriteLine("Load ************************");
```

Comment:
I will remove all of the debug statements in this file once the OpenShift deployment has been verified.